POC/api/main.py

```diff
@@ -1,63 +1,55 @@
-from fastapi import FastAPI, HTTPException
+from fastapi import FastAPI
 from fastapi.responses import HTMLResponse
-from sqlmodel import Field, Session, SQLModel, create_engine, select
+from sqlmodel import Field, SQLModel, create_engine
 from typing import Optional, Literal
-from fastui import FastUI, AnyComponent, prebuilt_html, components as c
-from fastui.components.display import DisplayMode, DisplayLookup
-from fastui.events import GoToEvent, BackEvent
+from fastui import FastUI, AnyComponent, prebuilt_html
 from pydantic import BaseModel, Field as DanticField
 
 app = FastAPI()
-fast_ui = FastUI(app)
 
 DATABASE_URL = "sqlite:///./test.db"
 engine = create_engine(DATABASE_URL)
 
-field_types = Literal["string", "integer", "float", "date", "datetime", "relationship", "email", "phone"]
+field_types = Literal[
+    "string", "integer", "float", "date", "datetime", "relationship", "email", "phone"
+]
 
-class CreationModelField(SQLModel, table=True):
```

Comment:
I removed the Creation Models for now. I will rethink and write these once User Models and FastUI responses work.